packages/truffle-decoder/lib/types/request.ts

```diff
@@ -0,0 +1,6 @@
+import BN from "bn.js";
+
+export interface DecoderRequest {
+  requesting: "storage";
```

Comment:
`requesting` strikes me as a bit weird. How about `type` or `kind`? (If you're worried about `type`, it's fine since it's not a keyword when used via object access, i.e. `request.type`)

Reply:
Sure, I could just go with `type`.  Actually I might want to redo this a bit anyway -- split each type of request into its own type, with DecoderRequest being the union of those types.  Sure, right now there's only one type, but soon there'll be two, so may as well be ready.